src/core/twitter.md

```diff
@@ -0,0 +1,22 @@
+# Twitter Guidelines
+
+The project runs the Twitter account [@rustlang](https://twitter.com/rustlang).
+The account is handled by a small team of volunteers.
+
+The account will mostly tweet links to the Rust blog and Rust Insiders blog.
+Additionally it will retweet:
+
+* links to blog posts about Rust, retweeting the original author if possible,
```

Comment:
nit: I'd normally not include "," at the end of bullet points. The bullet points themselves indicate the items are listed.